modules/runner/src/integrationTest/java/org/apache/ignite/internal/sql/engine/datatypes/varbinary/ItVarBinaryExpressionTest.java

```diff
@@ -67,24 +67,32 @@ public void testPositionExpressionWithDynamicParameter() {
                 .check();
     }
 
-    /** {@code LENGTH} expression. */
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-19668")
+    /** {@code LENGTH} and {@code OCTET_LENGTH} expression. */
     @Test
     public void testLengthExpression() {
         checkQuery("SELECT LENGTH(x'010203')")
-                .withParams(varBinary(new byte[]{2}))
+                .returns(3).check();
+
+        checkQuery("SELECT OCTET_LENGTH(x'010203')")
                 .returns(3).check();
     }
 
-    /** {@code LENGTH} expression. */
-    @Disabled("https://issues.apache.org/jira/browse/IGNITE-19668")
+    /** {@code LENGTH} and {@code OCTET_LENGTH} expression with dynamic params. */
     @Test
     public void testLengthExpressionWithDynamicParameter() {
-        checkQuery("SELECT LENGTH(?)")
+        checkQuery("SELECT OCTET_LENGTH(?)")
+                .withParams(varBinary(new byte[]{1, 2, 3}))
+                .returns(3).check();
+
+        checkQuery("SELECT OCTET_LENGTH(?)")
+                .withParams(varBinary(new byte[0]))
+                .returns(0).check();
+
+        checkQuery("SELECT LENGTH(CAST (? AS VARCHAR))")
                 .withParams(varBinary(new byte[]{1, 2, 3}))
                 .returns(3).check();
 
-        checkQuery("SELECT LENGTH(?)")
+        checkQuery("SELECT LENGTH(CAST (? AS VARCHAR))")
```

Comment:
why do you put CAST operator for last two cases?

Reply:
just to make sure that it works with cast too

Reply:
by "it" did you mean LENGTH?

Reply:
yes, cast from varbinary to varchar returns the expected result.

Reply:
well, and what is expected result in this case?